packages/sdk-js/src/mongrule.ts

```diff
@@ -15,26 +16,29 @@ const _regexCache: { [key: string]: RegExp } = {};
 // The top-level condition evaluation function
 export function evalCondition(
   obj: TestedObj,
-  condition: ConditionInterface
+  condition: ConditionInterface,
+  savedGroups: SavedGroupsValues
```

Comment:
should this be optional?
`savedGroups?: ...`

Reply:
I don't think so. It's required in all contexts where `condition` might include a group operation, so I think the prereqs and such were just special cases and the general case is that it should be required

Reply:
you have this line: `savedGroups = savedGroups || {};`. Should that be removed then?

Reply:
also by not making it optional, there is a small but real risk of temporarily breaking libs (such as `shared`) which depend on evalCondition having 2 args.

Reply:
No, that was a suggestion from Jeremy that some users might not be using typescript or might have `// @ts-ignore` statements that prevent them from automatically catching the new signature.
Do you think there's a risk of breaking the libs even with the change to the `package.json`s which swaps `^1.0.1` to `1.0.1`? I would think that specifying an exact version will keep things in a safe intermediary state until the new code is in place.